rep-0153.rst

```diff
@@ -0,0 +1,94 @@
+REP: 153
+Title: ROS distribution files
+Author: Dirk Thomas <dthomas@osrfoundation.org>
+Status: Final
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 27-Oct-2018
+Post-History: 27-Oct-2018
+
+
+.. contents::
+
+Abstract
+========
+This REP updates the specification of the ROS distribution files facilitated in
+the building, packaging, testing and documenting process.
+
+The intention is to annotate ROS version with their major version (``1`` or
+``2``) and avoid the need to externally define and update a classification of
+ROS distribution names as ROS 1 or ROS 2.
+
+This REP is a revised version of REP 143 [1]_.
```

Comment:
```suggestion
This REP is an extension to the distribution file format defined in REP 143 [1]_.
```

Reply:
Good point. Since the proposed line was too long I didn't accept the suggestion. I also remove "distribution" from the sentence since it might be confusing - because the new REP doesn't change the distribution file format - but only the index file format. See 18e89993ea042ab770f14552bb5999680874d128

Reply:
lgtm